### PR TITLE
upgrade version of groovy to 2.5.10 in LEP

### PR DESCRIPTION
**Pull request by saylenty**
Upgrade version of lep to 0.2.0

### CODE REVIEW

File: gradle/version.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import java.util.regex.Pattern
 
-version = "2.1.8"//detectSemVersion()
+version = detectSemVersion()
```
Comment (vkirichenko) [inline gradle/version.gradle]: leave this line, as it was

File: gradle.properties [hunk 2]
```diff
@@ -4,4 +4,5 @@ org.gradle.daemon=true
 app.group=com.icthh.xm.commons
 app.name=xm-commons
 app.description=XM^online common libs
+app.version=2.1.9
```
Comment (vkirichenko) [inline gradle.properties]: remove this

File: build.gradle [hunk 3]
```diff
@@ -30,6 +30,7 @@ apply plugin: 'io.codearte.nexus-staging'
 // project parameters
 project.group = property('app.group')
 project.description = property('app.description')
+project.version = property('app.version')
```
Comment (vkirichenko) [inline build.gradle]: remove this

File: gradle/version.gradle (hunk 1 above)
Comment (saylenty) [reply gradle/version.gradle]: done

File: build.gradle (hunk 3 above)
Comment (saylenty) [reply build.gradle]: done

File: gradle.properties (hunk 2 above)
Comment (saylenty) [reply gradle.properties]: done

**Pull request merged**